packages/test-middlewares/README.md

```diff
@@ -0,0 +1,42 @@
+# sample-botfuel-dialog-extrafeatures
```

Comment:
this should be test-extrafeatures or test-middlewares now that we can add as many as test directories as we want